### PR TITLE
PLT-4554 [FE]Sell a subscription

### PR DESCRIPTION
**Pull request by amnambiar**
https://input-output.atlassian.net/browse/PLT-4554

### CODE REVIEW

**Comment by RSoulatIOHK** [comment]
I think it still needs to have an actual effect when clicking on the different "pay" buttons to close PLT-4554. But it looks good. You should wait for the more expert review from Bogdan, this was just my 2 cents.

File: react-web/src/pages/subscription/Subscription.tsx
Comment (bogdan-manole) [inline react-web/src/pages/subscription/Subscription.tsx:5]: Shouldn't we get the descriptions from `GET /tiers`?

File: react-web/src/pages/subscription/components/PricingCard/PricingCard.tsx
Comment (bogdan-manole) [inline react-web/src/pages/subscription/components/PricingCard/PricingCard.tsx:4]: why `any` ?

File: react-web/src/pages/subscription/components/PricingCard/PricingCard.tsx
Comment (bogdan-manole) [inline react-web/src/pages/subscription/components/PricingCard/PricingCard.tsx:29]: do we need the comment?

**Review by bogdan-manole — CHANGES_REQUESTED** [review]

**Comment by amnambiar** [comment]
Closing this PR as most recent changes are included within https://github.com/input-output-hk/dapps-certification/pull/64

**Pull request closed without merging**
